e2e/fixtures/pages/ExploreProfilesPage.ts

```diff
@@ -1,6 +1,6 @@
 import { expect, type Page } from '@playwright/test';
 
-import { DEFAULT_EXPLORE_PROFILES_URL_PARAMS } from '../../config/constants';
```

Comment:
Changes here and there taken from https://github.com/grafana/explore-profiles/pull/136